src/modules/users/user_repository.ts

```diff
@@ -71,7 +71,11 @@ export class UserRepository {
 
   public findById = (id: string) => this.Query().where('users.id', id).first()
 
-  public destroy = (id: number) => this.Users().where('id', id).delete()
+  public destroy = (id: number) =>
+    this.Users()
+      .where('users.id', id)
+      .leftJoin('partners', 'partners.id', '=', 'users.partner_id')
+      .delete()
```

Comment:
this pr is meant to delete a related user on a partner?

Reply:
Thankss for information